metricflow/model/semantics/metric_lookup.py

```diff
@@ -123,35 +121,6 @@ def configured_input_measure_for_metric(self, metric_reference: MetricReference)
         else:
             assert_values_exhausted(metric.type)
 
-    def measures_for_metric(
```

Comment:
Oh yeah I think this was used somewhere else, and isn't anymore. Agree it makes more sense in the one place it's used, although making the DataflowPlanBuilder longer is a bit of a bummer.